app/src/main/java/com/example/app/logout.java

```diff
@@ -0,0 +1,20 @@
+package com.example.app;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
```

Comment:
Why haven't you used ung logout function ni firebase?

Reply:
Please check out the documentation for logout here
https://firebase.google.com/docs/auth/android/custom-auth